Source/BoundaryConditions/PML.cpp

```diff
@@ -1004,39 +1005,39 @@ PushPMLPSATDSinglePatch (
     std::array<std::unique_ptr<amrex::MultiFab>,3>& pml_E,
     std::array<std::unique_ptr<amrex::MultiFab>,3>& pml_B ) {
 
-    using Idx = SpectralPMLIndex;
+    using SpIdx = SpectralPMLIndex;
```

Comment:
I changed `Idx` to `SpIdx` here, in order to make it clear that this enum corresponds to the fields in spectral space, while `PMLComp` corresponds to the fields in real space.